src/particle/converters/lhcb.py

```diff
@@ -0,0 +1,56 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2018-2021, Eduardo Rodrigues and Henry Schreiner.
+#
+# Distributed under the 3-clause BSD license, see accompanying file LICENSE
+# or https://github.com/scikit-hep/particle for details.
+
+from __future__ import absolute_import
+
+from .. import data
+from ..pdgid import PDGID
+from .bimap import BiMap, DirectionalMaps
+
+LHCbName2PDGIDBiMap = BiMap(
+    PDGID,
+    str,
+    converters=(int, str),
+    filename=data.basepath / "pdgid_to_lhcbname.csv",
+)
+LHCbName2PDGIDBiMap.__doc__ = """
+Bi-bidirectional map between PDG IDs and LHCb names.
+
+Examples
+--------
+>>> name = LHCbName2PDGIDBiMap[PDGID(22)]
+>>> name
+'gamma'
+
+>>> pdgid = LHCbName2PDGIDBiMap['gamma']
+>>> pdgid
+<PDGID: 22>
+"""
+
+
+PDG2LHCbNameMap, LHCb2PDGNameMap = DirectionalMaps("PDGName", "LHCbName")
+
+PDG2LHCbNameMap.__doc__ = """
+Directional map between PDG and LHCb names.
+
+Examples
+--------
+>>> PDG2LHCbNameMap['J/psi(1S)']
+'J/psi'
```

Comment:
This I don't understand: the LHCb name is also J/psi(1S).

Reply:
Lazy copy/paste from the EvtGen version, my bad.

Reply:
It happens. Best is to provide an example where the 2 names differ.